typescript/sdk/src/token/Token.ts

```diff
@@ -161,6 +166,28 @@ export class Token implements IToken {
         sourceChannel: 'channel-0',
         type: TokenConnectionType.Ibc,
       });
+    } else if (standard === TokenStandard.EvmIntent) {
+      assert(
+        this.intentRouterAddressOrDenom,
```

Comment:
Is this still required? I thought we said we would overload addressOrDenom for the router and collateralAddressorDenom for the token?